src/main/java/com/jcabi/dynamo/mock/MkTable.java

```diff
@@ -100,4 +100,16 @@ public Region region() {
     public String name() {
         return this.self;
     }
+
+    /**
+     * Delete item from aws table.
+     * @todo Implement #25
+     * @param attributes Attributes
+     * @throws IOException In case of DynamoDB failure
+     */
```

Comment:
actually, no
your task considered to be done when delete operation is implemented in ```AwsTable```, ```MkTable``` and ```ReTable``` with unit tests for all of them.
Obviously, you can't do this in 30 min, that's why you have to create a puzzles